manager/src/main/java/org/openremote/manager/app/ConfigurationService.java

```diff
@@ -113,4 +115,13 @@ public void saveImageFile(String path, FileInfo fileInfo) {
         }
 
     }
+
+    public Optional<File> getManagerConfig(){
+        File file = persistenceService.getStorageDir().resolve("manager").resolve("manager_config.json").toFile();
+
+        if(file.exists() && !file.isDirectory()){
+           return Optional.of(file);
+        }
+        return Optional.empty();
+    }
```

Comment:
Should fallback to look in `OR_CUSTOM_APP_DOCROOT` if doesn't exist in storageDir so that it can be provided in the deployment image if desired.

Reply:
This is still outstanding

Reply:
This has been changed as we now cache the manager, and when we do get the manager, and I do this in `getManagerConfigFile`.